samples/led/main.c

```diff
@@ -1,22 +1,17 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
+  // Set front LEDs to shine green and then red.
+  // For information about timings and such, see
+  // https://www.XboxDevWiki.net/LED
```

Comment:
Same as above; however, I wouldn't even bother documenting this as detailed here.

People using the API will probably look this up themselves.